python/tests/test_eval.py

```diff
@@ -23,13 +53,17 @@ def test_metrics():
 
 @pytest.mark.asyncio
 async def test_run_with_inputs_sanity_check():
-    """No easy way to mock LLM calls,
-    so just test the imports and sanity check output."""
+    """No easy way to mock LLM calls from outside run_test_suite_with_inputs.
+
+    Instead, give empty list and just test the imports and sanity check output."""
```

Comment:
Give an actual example on how to call this (or do GH link example to code that does this)

Reply:
I'm confused what the ask was here @rossdanlm